dashboard/app/models/curriculum_reference.rb

```diff
@@ -0,0 +1,42 @@
+# == Schema Information
+#
+# Table name: levels
+#
+#  id                       :integer          not null, primary key
+#  game_id                  :integer
+#  name                     :string(255)      not null
+#  created_at               :datetime
+#  updated_at               :datetime
+#  level_num                :string(255)
+#  ideal_level_source_id    :integer
+#  solution_level_source_id :integer
+#  user_id                  :integer
+#  properties               :text(65535)
+#  type                     :string(255)
+#  md5                      :string(255)
+#  published                :boolean          default(FALSE), not null
+#  notes                    :text(65535)
+#
+# Indexes
+#
+#  index_levels_on_game_id  (game_id)
+#  index_levels_on_name     (name)
+#
+
+class CurriculumReference < External
+  def dsl_default
+    <<-TEXT.strip_heredoc.chomp
+    # Update the following lines to give you level a unique name and point at
+    # the appropriate url on docs.code.org
+    name '#{name || 'unique level name here'}'
+    reference '/csd/maker_leds/index.html'
+    TEXT
+  end
+
+  # Get the URL of the studio.code.org/docs routes (that serves as a proxy to
+  # our docs.code.org route)
```

Comment:
https://github.com/code-dot-org/code-dot-org/pull/13775#issuecomment-287206887

> We're not stripping JS/CSS from the proxied page?

The frame is same-domain, so we're implicitly trusting this content.  I thought the plan was to treat it as basically untrusted?  https://codedotorg.slack.com/archives/dashboard-dev/p1489103320000580